nekoyume/Assets/_Scripts/BlockChain/ActionRenderHandler.cs

```diff
@@ -470,14 +470,25 @@ private void ResponseSell(ActionBase.ActionEvaluation<Sell> eval)
             {
                 var avatarAddress = eval.Action.sellerAvatarAddress;
                 var tradableId = eval.Action.tradableId;
-
-                LocalLayerModifier.AddItem(avatarAddress, tradableId);
-                var format = L10nManager.Localize("NOTIFICATION_SELL_COMPLETE");
-
+                var blockIndex = Game.Game.instance.Agent.BlockIndex;
+                var count = eval.Action.count;
                 var avatarState = new AvatarState((Bencodex.Types.Dictionary) eval.PreviousStates.GetState(avatarAddress));
-                if (avatarState.inventory.TryGetNonFungibleItem(tradableId, out var item))
+                if (avatarState.inventory.TryGetTradableItems(tradableId, blockIndex, count, out var item))
                 {
-                    UI.Notification.Push(MailType.Auction, string.Format(format, item.item.GetLocalizedName()));
+                    string message = string.Empty;
+                    if (count > 1)
```

Comment:
How about telling you in the form of `I'll cancel the sale of {0}({1})` even when you buy one `IFungibleItem`?
To do this, you can use a method of checking whether `items.First().Item` is `IFungibleItem` instead of `count`.
```suggestion
                    if (items.First().Item is IFungibleItem)
```

Reply:
> How about telling you in the form of `I'll cancel the sale of {0}({1})` even when you buy one `IFungibleItem`?
> To do this, you can use a method of checking whether `items.First().Item` is `IFungibleItem` instead of `count`.

It was decided to separate notifications  by design.